utils/experiment_options/experimentoptions.py

```diff
@@ -100,59 +96,42 @@ def __init__(self):
-        lines[:] = [line for line in lines if int(line.split()[1]) > today.year or
-                    (int(line.split()[1]) == today.year and float(line.split()[2]) >
-                     year_timedelta.total_seconds())]  # only take present & future hdw data
-
-        # there should only be one line left, however if there are more we will take the
-        # one that expires first.
-        if len(lines) > 1:
-            times = [[line.split()[1], line.split()[2]] for line in lines]
-            min_year = times[0][0]
-            min_yrsec = times[0][1]
-            hdw_index = 0
-            for i in range(len(times)):
-                year = times[i][0]
-                yrsec = times[i][1]
-                if year < min_year:
-                    hdw_index = i
-                elif year == min_year:
-                    if yrsec < min_yrsec:
-                        hdw_index = i
-            hdw = lines[hdw_index]
-        else:
-            try:
-                hdw = lines[0]
-            except IndexError:
-                errmsg = 'Cannot find any valid lines for this time period in the hardware file ' \
-                         '{}'.format((hdw_dat_file + self.site_id))
-                raise ExperimentException(errmsg)
+
+        # Take the final line
+        try:
+            hdw = lines[-1]
+        except IndexError:
+            errmsg = 'Cannot find any valid lines in the hardware file: ' \
+                     '{}'.format((hdw_dat_file + self.site_id))
+            raise ExperimentException(errmsg)
         # we now have the correct line of data.
 
         params = hdw.split()
-        if len(params) != 19:
-            errmsg = 'Found {} parameters in hardware file, expected 19'.format(len(params))
+        if len(params) != 22:
+            errmsg = 'Found {} parameters in hardware file, expected 22'.format(len(params))
             raise ExperimentException(errmsg)
 
-        self._geo_lat = params[3]  # decimal degrees, S = negative
-        self._geo_long = params[4]  # decimal degrees, W = negative
-        self._altitude = params[5]  # metres
-        self._boresight = params[6]  # degrees from geographic north, CCW = negative.
-        self._beam_sep = params[7]  # degrees TODO is this necessary, or is this a min. - for
-        # post-processing software in RST? check with others.
+        self._status = params[1]  # 1 operational, -1 offline
+        self._geo_lat = params[4]  # decimal degrees, S = negative
+        self._geo_long = params[5]  # decimal degrees, W = negative
+        self._altitude = params[6]  # metres
+        self._boresight = params[7]  # degrees from geographic north, CCW = negative.
+        self._boresight_shift = params[8]  # degrees from physical boresight. nominal 0.0 degrees
+        self._beam_sep = params[9]  # degrees, nominal 3.24 degrees
         self._velocity_sign = params[8]  # +1.0 or -1.0
```

Comment:
Is it correct that `self._velocity_sign` is the same value as `self._boresight_shift`?

Reply:
No, good catch here. Line 121 should be `self._velocity_sign = params[10]`

Reply:
Luckily, this value isn't used for anything in Borealis, so there isn't any collateral from having this wrong for the past couple months.